backend/analytics_server/mhq/store/repos/code.py

```diff
@@ -328,6 +328,25 @@ def get_team_repos(self, team_id) -> List[OrgRepo]:
 
         team_repo_ids = [tr.org_repo_id for tr in team_repos]
         return self.get_repos_by_ids(team_repo_ids)
+    
+    @rollback_on_exc
+    def get_prs_not_reviewed_merged(self,team_id):
+        AllOrg = self.get_team_repos(team_id)
+        AllOrg_ids = [tr.id for tr in AllOrg]
+        print(AllOrg)
+        AllPullRequestEventReviweinOrgIds = (self._db.session.query(PullRequestEvent)
+                                  .filter(PullRequestEvent.org_repo_id.in_(AllOrg_ids))
+                                  .all()
+                                  )
+        ListofReviwedPrs = [ tr.pull_request_id for tr in AllPullRequestEventReviweinOrgIds]
+        AllPullRequestMergedNotReviwed = (self._db.session.query(PullRequest)
+                                          .filter(PullRequest.id.not_in(ListofReviwedPrs))
+                                          .filter(PullRequest.state == PullRequestState.MERGED)
+                                          .all())
+        print(AllPullRequestMergedNotReviwed)
+        
+        return {"countofMergedReviewedPrs": len(AllPullRequestEventReviweinOrgIds),"countOfMergedNOtReviwedPrs":len(AllPullRequestMergedNotReviwed)}
```

Comment:
The queries can be way simpler. So, a Team can be Mapped to Repos via TeamRepo and PullRequests have a foreign key to relationship to OrgRepos.

Reply:
To find a PullRequest merged without review, we can check if a PRs merge time is null. Merge time is the time taken from approval to when the PR was merged. If the PR is merged without approval, merge is null. This is an internal detail which you can figure out by reading this code: https://github.com/middlewarehq/middleware/blob/main/backend/analytics_server/mhq/service/code/sync/etl_code_analytics.py.